engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -122,13 +122,13 @@ angular.module('Bastion.capsule-content').controller('CapsuleContentController',
 
         $scope.isTaskInProgress = isTaskInProgress;
 
-        $scope.syncCapsule = function () {
+        $scope.syncCapsule = function (skipMetadataCheck = false) {
             if (!$scope.syncState.is(syncState.SYNCING)) {
 
                 $scope.syncErrorMessages = [];
                 $scope.syncState.set(syncState.SYNC_TRIGGERED);
 
-                CapsuleContent.sync({id: capsuleId}).$promise.then(function (task) {
+                CapsuleContent.sync({id: capsuleId, skip_metadata_check: skipMetadataCheck}).$promise.then(function (task) {
```

Comment:
@waldenraines this *works*, but I am not sure if am going about things the right way here, can you take a look?